server/database/dbconnection.js

```diff
@@ -1,17 +1,16 @@
-const { Pool } = require("pg");
-require("env2")("./config.env");
+const pgp = require('pg-promise')();
+require('env2')('./config.env');
 
-let { DEV_DB_URL } = process.env;
-if (process.env.NODE_ENV === "test") {
-  DB_URL = process.env.TEST_DB_URL;
-}
+let { DB_URL } = process.env;
+const { NODE_ENV } = process.env;
+const { TEST_DB_URL } = process.env;
```

Comment:
oyy you could just do `{DB_URL, NODE_ENV, TEST_DB_URL } = process.env` and still would work the same

Reply:
Good point. updated.